@xen-orchestra/backups/Backup.js

```diff
@@ -55,20 +55,45 @@ exports.Backup = class Backup {
   }
 
   async _runMetadataBackup() {
-    const config = this._config
-    const job = this._job
     const schedule = this._schedule
+    if (schedule === undefined) {
+      throw new Error('backup job cannot run without a schedule')
+    }
```

Comment:
Are there cases where this could happen?

IMHO, it's not supported at all by `@xen-orchestra/backups/Backup`.

Reply:
I don't know why we did this condition in the metadata/VM backup.

How we can know if it's supported by `@xen-orchestra/backups/Backup.` or not? Thank you :)

Reply:
> I don't know why we did this condition in the metadata/VM backup.

Let's remove it then.

> How we can know if it's supported by `@xen-orchestra/backups/Backup.` or not?

It's a design choice, even if we did not properly enforced it (by an `assert`), it was always implicit that it was not optional, for instance, we did not check existence before `schedule.id`.